docs/bash.md

```diff
@@ -887,6 +1079,160 @@ const {pid} = $`npm run build`;
 
 [More info.](termination.md#inter-process-termination)
 
+### CLI arguments
+
+```js
+// zx
+const {myCliFlag} = argv;
+```
+
+```js
+// Execa
+import minimist from 'minimist';
```

Comment:
I would go with https://nodejs.org/api/util.html#utilparseargsconfig

Reply:
Fixed in aa3fca9c